docs/sources/tempo/release-notes/v2-1.md

```diff
@@ -0,0 +1,128 @@
+---
+title: V2.1
+weight: 55
+---
+
+# Version 2.1 release notes
+
+The Tempo team is pleased to announce the release of Tempo 2.1.
+
+With this release, Tempo enhances TraceQL so you can select traces based on kind, basic arithmetic functions, average/count, and numbers. In addition, Tempo 2.1 introduces vParquet2, a new experimental Parquet version designed to be more compatible with other Parquet implementations.
+
+>**Note**: For a complete list of changes and enhancements, refer to the [Tempo 2.1 changelog](https://github.com/grafana/tempo/releases).
+
+## Features and enhancements
+
+Some of the most important features and enhancements in Tempo 2.1 are highlighted below.
+
+### Enhancements to TraceQL
+
+Inspired by PromQL and LogQL, TraceQL is a query language designed for selecting traces in Tempo. TraceQL makes it possible to find traces that otherwise would have been difficult or even impossible to identify with Tempo’s 1.x search.
+
+With this release, we’ve expanded the TraceQL language. In Tempo 2.0, TraceQL allowed users to select traces based upon resource attributes, time, duration, and basic aggregates. In Tempo 2.1, users can also select traces based on the following:
+
+* Enable querying by `kind`, an intrinsic attribute  (PR [2217](https://github.com/grafana/tempo/pull/2217))
+    * ``{ kind = server }`, `{ kind = client }`, `{ kind = consumer }``
+    * ``{ kind = server && duration > 1s }``
+* Support arithmetic expressions (PR [2146](https://github.com/grafana/tempo/pull/2146))
+    * ``{ span.bytesProcessed > 10 * 1024 * 1024 }``
+    * ``{ span.bytesProcessed / 1024 / 1024 > 10 }``
+    * ``{ span.jobs / span.bytes > 3 }``
+* Enable min, max, and sum aggregate filters
+    * ``{} | min(duration) > 1s``
+    * ``{} | max(span.bytesProcessed) &lt; 1024 * 1024``
+    * ``{} | sum(duration) > 1h``
+* Support for more numeric types: Compare floating point numbers and integers, allow floating point numbers for durations, and support negative numbers when doing aggregate filters. (PR [2139](https://github.com/grafana/tempo/pull/2139), [2304](https://github.com/grafana/tempo/pull/2304), [2289](https://github.com/grafana/tempo/pull/2289))
+    * ``{ 1 = 1.0 }` `
+    * ``{ duration > 1.5s }`, `{duration > 1s500ms }``
+    * ``{} | count() > -1``
+
+This update also includes major performance improvements to TraceQL query performance.
+
+To learn more about the TraceQL syntax, see the [TraceQL documentation](http://localhost:3002/docs/tempo/latest/traceql/). For information on planned future extensions to the TraceQL language, see [future work](http://localhost:3002/docs/tempo/latest/traceql/architecture/).
+
+
+### TraceQL query editor improvements (experimental)
+
+Starting in Grafana 9.5, users have access to a Tempo Search tab powered by TraceQL. To enable this feature, use Grafana’s ``traceqlSearch` `feature flag.  The editor can autocomplete queries and has a streamlined interface.
```

Comment:
i think double backticks are impacting a lot of formatting in this section

Reply:
I'll fix it. Stupid thing is stupid.